app/assets/stylesheets/application.css.scss

```diff
@@ -178,7 +162,7 @@ body h1 h2 h3 h4 {
 }
 
 .nav_custom {
-  background-color: #E66C2C;
+  background-color: rgba(20, 20, 20, 0.0);
```

Comment:
Color literals like `rgba(20, 20, 20, 0)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>`0.0` should be written without the mantissa as `0`